.github/workflows/publish-docs.yml

```diff
@@ -0,0 +1,71 @@
+name: Publish docs to GitHub pages
+
+on:
+  push:
+    branches:
+      - 'docs/**'
+    paths:
+      - 'docs/**'
+
+jobs:
+  publish:
+    runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        python-version: [3.7]
+        package: [docs]
+
+    steps:
+      - uses: actions/checkout@v2
+        with:
+          # Only a single commit is fetched by default, for the ref/SHA that triggered the workflow.
+          # Set fetch-depth: 0 to fetch all history for all branches and tags.
+          fetch-depth: 0
+      - name: Set up Python ${{ matrix.python-version }}
+        uses: actions/setup-python@v2
+        with:
+          python-version: ${{ matrix.python-version }}
+      - name: Set the current branch environment variable
+        working-directory: ${{ matrix.package }}
+        run: |
+          CURRENT_BRANCH_VAR=${GITHUB_REF#refs/heads/}
+          echo "Current branch: $CURRENT_BRANCH_VAR"
+          # Set CURRENT_BRANCH environment variable to the current branch name.
+          echo "::set-env name=CURRENT_BRANCH::$(echo $CURRENT_BRANCH_VAR)"
+      - name: Display all remote branches
+        working-directory: ${{ matrix.package }}
+        run: |
+          # Display all remote branches
+          git branch -r
+      - name: Set the latest docs branch environment variable
+        working-directory: ${{ matrix.package }}
+        run: |
+          # Get only docs branches, extract the "docs/x.y.z" part, sort them in descending order, and get the first one.
+          LATEST_DOCS_BRANCH_VAR=$(git branch -r | grep -e ".*\/*docs\/[0-9].[0-9].[0-9]" | sed -n "s/.*\/*\(docs\/[0-9].[0-9].[0-9]\).*/\1/p" | sort -r | head -n 1)
+          echo "Latest docs branch: $LATEST_DOCS_BRANCH_VAR"
+          # Set the LATEST_DOCS_BRANCH environment variable.
+          echo "::set-env name=LATEST_DOCS_BRANCH::$(echo $LATEST_DOCS_BRANCH_VAR)"
+      - name: Check that the current branch is the latest docs branch, fail otherwise
```

Comment:
can we add instructions to the `README` or somewhere else on how to use this action to publish docs changes using your own fork, and what to name the `docs` branch etc. locally?